src/visip/dev/parameters.py

```diff
@@ -142,11 +151,11 @@ def extract_func_signature(func, skip_self=True):
 
         if param.kind == param.VAR_POSITIONAL:
             assert default == ActionParameter.no_default
-            param = ActionParameter(None, annotation, default)
```

Comment:
type_hint unwrapping and conversion should be made as soon as possible directly in _extract_signature.